beta/tests/tensorflow/data/ac_configs/retinanet_int8_w_sym_t_a_sym_t.yml

```diff
@@ -6,17 +6,17 @@ models:
         adapter:
           type: retinanet_tf2
           boxes_outputs:
-            - retinanet/nncf_wrapper_box-predict/BiasAdd_4/Add
-            - retinanet/nncf_wrapper_box-predict/BiasAdd_3/Add
-            - retinanet/nncf_wrapper_box-predict/BiasAdd_2/Add
-            - retinanet/nncf_wrapper_box-predict/BiasAdd_1/Add
-            - retinanet/nncf_wrapper_box-predict/BiasAdd/Add
+            - retinanet/box-predict/BiasAdd_4/Add
+            - retinanet/box-predict/BiasAdd_3/Add
+            - retinanet/box-predict/BiasAdd_2/Add
+            - retinanet/box-predict/BiasAdd_1/Add
+            - retinanet/box-predict/BiasAdd/Add
           class_outputs:
-            - retinanet/nncf_wrapper_class-predict/BiasAdd_4/Add
-            - retinanet/nncf_wrapper_class-predict/BiasAdd_3/Add
-            - retinanet/nncf_wrapper_class-predict/BiasAdd_2/Add
-            - retinanet/nncf_wrapper_class-predict/BiasAdd_1/Add
-            - retinanet/nncf_wrapper_class-predict/BiasAdd/Add
+            - retinanet/class-predict/BiasAdd_4/Add
+            - retinanet/class-predict/BiasAdd_3/Add
+            - retinanet/class-predict/BiasAdd_2/Add
+            - retinanet/class-predict/BiasAdd_1/Add
+            - retinanet/class-predict/BiasAdd/Add

```

Comment:
Hello! After changes we don't need retinanet_int8_w_sym_t_a_sym_t.yml  anymore. It's will be created in test on-the-fly, but for this need modification test_sota_checkpoints.py

Reply:
Deleted